server/routes/user.py

```diff
@@ -85,7 +86,18 @@ def index():
     return "Data Commons </br> <a href='/user/auth/login'><button>Login</button></a>"
 
 
-@bp.route('/upload/import', methods=['POST'])
+@bp.route('/imports')
+@login_is_required
+def get_imports() -> Dict[str, Dict]:
+    user_id = libutil.hash_id(session['oauth_user_id'])
+    user = user_api.get_user(user_id)
+    imports: Dict[str, Dict] = {}
+    for im in user.reference.collection(user_api.IMPORT_COLLECTION).stream():
+        imports[im.id] = im.to_dict()
```

Comment:
Should this be in a helper in user.py (corollary to add_import, so all interaction with the db happens in that module).